src/main/java/com/scalefocus/blog_api/mapper/BlogMapper.java

```diff
@@ -0,0 +1,31 @@
+package com.scalefocus.blog_api.mapper;
+
+
+import com.scalefocus.blog_api.dto.BlogDto;
+import com.scalefocus.blog_api.entity.Blog;
+
+public class BlogMapper {
```

Comment:
Static is ok, but is better to use the mapper as @Component. When we use Spring this way is more aligned with its approach. Also dependency injection and testing is easer.

Reply:
Done